src/validations/member.js

```diff
@@ -32,18 +46,32 @@ const validateCreate = (req, res, next) => {
 
 const validateUpdate = (req, res, next) => {
   const updateMemberValidate = Joi.object({
-    firstName: Joi.string().min(3).max(20).pattern(/^[a-zA-Z]+(?: [a-zA-Z]+)?$/),
-    lastName: Joi.string().min(3).max(20).pattern(/^[a-zA-Z]+(?: [a-zA-Z]+)?$/),
+    firstName: Joi.string().min(3).max(20).pattern(/^[a-zA-Z]+(?: [a-zA-Z]+)?$/)
+      .messages({
+        'string.pattern.base': 'First name must be only made of letters(it can be a compound Name)',
+      }),
+    lastName: Joi.string().min(3).max(20).pattern(/^[a-zA-Z]+(?: [a-zA-Z]+)?$/)
+      .messages({
+        'string.pattern.base': 'Last name must be only made of letters(it can be a compound Name)',
+      }),
     dni: Joi.number().min(1000000).max(99999999).integer(),
     phone: Joi.number().min(1000000000).max(9999999999).integer(),
-    email: Joi.string().email().min(5).max(30)
-      .regex(/^[^@]+@[^@]+.[a-zA-Z]{2,}$/)
+    email: Joi.string().regex(/^(?!\.)(?!.*\.\.)[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(\.[a-z-]+)*(\.[a-z]{2,4})$/).min(5)
+      .messages({
+        'string.pattern.base': 'Email is not valid, muy contain only one @ and a valid domain',
+      })
       .lowercase(),
-    city: Joi.string().min(2).max(20).regex(/^[a-zA-Z]+(?:[\s-][A-Za-z]+)*$/),
+    city: Joi.string().min(2).max(20).regex(/^[a-zA-Z]+(?:[\s-][A-Za-z]+)*$/)
+      .messages({
+        'string.pattern.base': 'Password must contain at least 8 characters and cannot contain blank spaces',
```

Comment:
this is for password, this is city message
![Captura de pantalla 2023-06-13 104602](https://github.com/BaSP-m2023/juvi-megarocket-server/assets/127536596/f6676b83-552d-48e5-9805-29131947a544)